integration_testing/features/coach/coach-create-group.feature

```diff
@@ -14,6 +14,15 @@ Feature: Coach create groups
     Then the modal closes
       And I see the new group on *Class groups* page
 
+  Scenario: Group name does not pass validation
+    Given that I have entered characters over the character limit
+    # This seems to be blocked, not possible to go over the limit
+      Or I entered an improper character such as a symbol
+      # This is not implemented
+      Or I left the name field empty
+      # This seems the only error that triggers the red highlight
+    Then I see the text field highlighted in red
```

Comment:
@indirectlylit see comments `#` above ^^

Reply:
> I have entered characters over the character limit

This one should be phrased like "There is a limit to how many characters I can enter"

> Or I entered an improper character such as a symbol

I don't think this is an accurate constraint - any character should be allowed

>       Or I left the name field empty
>       # This seems the only error that triggers the red highlight

Entering the name of a group that already exists should also trigger a validation warning

Reply:
> any character should be allowed

Yes, any character is allowed, actually... Will eliminate this, and add the existing name constraint!